src/App.jsx

```diff
@@ -6,19 +7,24 @@ import Questions from "./pages/Questions";
 import Quizzes from "./pages/Quizzes";
 import Result from "./pages/Result";
 import Signup from "./pages/Signup";
```

Comment:
_You could export these into an index file. _

Reply:
yeah will do it after submission thanks for PR